### ISSUE TITLE
Slight improvement to accuracy score

### ISSUE THREAD

**Issue by apavlo89** [WarrierRajeev/UFC-Predictions#19]
Hello there,

I think I increased prediction accuracy (using 80%-20% split) ever so slightly using TPOT (no oversampling applied yet).

try this:

```
# Average CV score on the training set was: 0.6958245897228948
exported_pipeline = make_pipeline(
    make_union(
        make_pipeline(
            StackingEstimator(estimator=BernoulliNB(alpha=0.001, fit_prior=False)),
            ZeroCount()
        ),
        FunctionTransformer(copy)
    ),
    StackingEstimator(estimator=SGDClassifier(alpha=0.01, eta0=0.1, fit_intercept=False, l1_ratio=0.0, learning_rate="constant", loss="perceptron", penalty="elasticnet", power_t=0.1)),
    MaxAbsScaler(),
    XGBClassifier(learning_rate=0.1, max_depth=2, min_child_weight=19, n_estimators=100, n_jobs=1, subsample=0.4, verbosity=0)
)
# Fix random state for all the steps in exported pipeline
set_param_recursive(exported_pipeline.steps, 'random_state', 2)

exported_pipeline.fit(training_features, training_target)
results = exported_pipeline.predict(testing_features)

```

**Comment by apavlo89** [comment]
The accuracy score might increase even more as the optimisation code is still running on my unis cluster computer  :P . I'll be updating main post with updated best

**Comment by apavlo89** [comment]
I also created a new variable called B_finish_decision_ratio (also for R_) where I sum all the decision finish columns / finish columns. This improves the score. Keep in mind when its 0/0 i just turn it into a 0 value

**Comment by MathemAddict** [comment]
> The accuracy score might increase even more as the optimisation code is still running on my unis cluster computer :P . I'll be updating main post with updated best

What accuracy did you end up with?

**Comment by jjphung** [comment]
@apavlo89 what do you mean by "finish columns"? I wasn't able to find features named "finish".
